docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md

```diff
@@ -772,6 +772,49 @@ When set, the instrumentation starts with [Test Orchestrator version 1.1.1](http
 ```
 ---
 
+### `appSettings`
+<p><small>| OPTIONAL | OBJECT |</small></p>
+
+Application settings for Real Device test.
```

Comment:
Plural here:
```suggestion
Application settings for Real Device tests.
```

Hm. I think it's best if we keep the casing consistent across the use of `real device / Real Device`. See further instances below where it's different.